src/app/grid/grid-multi-row-layout-configuration/grid-multi-row-layout-configuration.component.scss

```diff
@@ -20,7 +20,7 @@
 }
 
 .layoutListContainer {
-    width: fit-content;
+    display: fit-content;
```

Comment:
There is no `fit-content` value for the `display` property. Probably should be removed ?